install.sh

```diff
@@ -145,8 +145,8 @@ case "$OSTYPE" in
 esac
 
 # TODO: Check which is in use
-echo 'eval "$(atuin init zsh)"' >> ~/.zshrc
+echo '\neval "$(atuin init zsh)"' >> ~/.zshrc
```

Comment:
This line doesn't work for me.  It adds the literal line `\neval "$(atuin init zsh)"` to the end of my `~/.zshrc`, and it doesn't add a newline.

Shellcheck warning [SC2028](https://github.com/koalaman/shellcheck/wiki/SC2028) recommends to use [`printf`](https://manned.org/printf.1) instead of echo, for this very reason.

```suggestion
printf '\neval "$(atuin init zsh)"' >> ~/.zshrc
```
I tested that, and it works! 🎉

FYI—my bash and OS version are the following:
```console
$ bash --version
GNU bash, version 5.1.8(1)-release (x86_64-apple-darwin20.3.0)
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
$ sw_vers
ProductName:	macOS
ProductVersion:	11.3.1
BuildVersion:	20E241
```

Reply:
I was going to suggest the same thing. Echo with newlines is not always supported